product_import_ubl/tests/test_ubl_catalogue_import.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+from odoo.tools import mute_logger
+
+from .common import get_test_data
+
+
+class TestUblOrderImport(SavepointCase):
+
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
```suggestion
        # cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
uncomment this unless you need tracking features